src/user/user.action.js

```diff
@@ -201,6 +202,30 @@ export const getRepositories = user => {
   };
 };
 
+export const getStarredRepositories = user => {
+  return (dispatch, getState) => {
+    const url = `/users/${user.login}/starred?&sort=updated&per_page=50`;
```

Comment:
Let's remove the sorting by `updated` (remove `sort=updated`), then the default sorting will work according to the date of creation (`created`), it is this kind of sorting that is used on GitHub right now.